dev-libs/efl/efl-1.20.7.ebuild

```diff
@@ -0,0 +1,326 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+MY_P=${P/_/-}
+
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_SUB_PROJECT="core"
+	EGIT_URI_APPEND="${PN}"
+elif [[ *"${PV}" == *"_pre"* ]] ; then
+	MY_P=${P%%_*}
+	SRC_URI="https://download.enlightenment.org/pre-releases/${MY_P}.tar.xz"
+else
+	SRC_URI="https://download.enlightenment.org/rel/libs/${PN}/${MY_P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~hppa ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x64-solaris ~x86-solaris"
+fi
+
+inherit enlightenment pax-utils gnome2-utils xdg-utils
```

Comment:
To be honest, this eclass is so horrible I would suggest putting it on your list to stop using it. Then, you could also bump to EAPI=6.

Reply:
Should I be calling the binary in the ebuild to run the update myself ?
If gnome2-utils and xdg-utils eclasses are broken, maybe we could fix them